### PR TITLE
Add repo setup

### PR DESCRIPTION
**Pull request by damacus**

### CODE REVIEW

**Review by rshade — APPROVED** [review]

File: repository-setup.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# Repository Layout
+
+- Master is always Protected.
```
Comment (tas50) [inline repository-setup.md]: We should elaborate here since there's several options for protected master.

Comment (rshade) [reply repository-setup.md]: I guess maybe a screenshot of haproxy?

File: repository-setup.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+- Review where possible must have a second pair of eyes.
+- Squash merging is up to the repository admin
```
Comment (josephholsten) [inline repository-setup.md]: s/admin/primary maintainer/ ?

File: repository-setup.md (hunk 1 above)
Comment (josephholsten) [reply repository-setup.md]: https://cl.ly/3u043G0x363M

**Review by josephholsten — APPROVED** [review]
I'm comfortable with this as-is, but if you want to address the comments that would be even better!

**Review by josephholsten — APPROVED** [review]
LGTM

File: repository-setup.md [hunk 3]
```diff
@@ -0,0 +1,9 @@
+# Repository Layout
+
+- Master is always Protected.
+- Review where possible must have a second pair of eyes.
+- Squash merging is up to the repository primary maintainer
+
+We aim to have clean understandable updates, and good git history. Rather than hiding a mess under the carpet.
```
Comment (tas50) [inline repository-setup.md]: Make this a 2nd sentence on the last item in the list. It's a supporting note for that specific item and not all the items

File: repository-setup.md [hunk 4]
```diff
@@ -0,0 +1,9 @@
+# Repository Layout
+
+- Master is always Protected.
```
Comment (tas50) [inline repository-setup.md]: Protected probably doesn't need to be capitalized

File: repository-setup.md [hunk 5]
```diff
@@ -0,0 +1,9 @@
...
+- Review where possible must have a second pair of eyes.
+- Squash merging is up to the repository primary maintainer
```
Comment (tas50) [inline repository-setup.md]: Merge strategy (squash vs. merge vs. rebase) is up to the repository's primary maintainer

File: repository-setup.md [hunk 6]
```diff
@@ -0,0 +1,8 @@
+# Repository Layout
+
+* Master is always protected.
+* Review where possible must have a second pair of eyes.
+* Squash merging is up to the repository primary maintainer
+  * We aim to have clean understandable updates, and good git history. Rather than hiding a mess under the carpet.
```
Comment (zuazo) [inline repository-setup.md:6]: I think you need to add an empty line here. If you see the [View](https://github.com/sous-chefs/meta/blob/71e30a9968fca7eb89a59fb67420babea6bf7683/repository-setup.md), the `# Example` heading is not interpreted correctly.

File: repository-setup.md [hunk 7]
```diff
@@ -0,0 +1,8 @@
...
+# Example
```
Comment (tas50) [inline repository-setup.md:8]: Missing space is breaking the formatting

**Review by tas50 — APPROVED** [review]

**Review by zuazo — CHANGES_REQUESTED** [review]

**Review by zuazo — APPROVED** [review]

**Pull request merged**